lib/ExternalModuleFactoryPlugin.js

```diff
@@ -38,9 +39,10 @@ ExternalModuleFactoryPlugin.prototype.apply = function(normalModuleFactory) {
 						return handleExternal(dependency.request, callback);
 					}
 				} else if(Array.isArray(externals)) {
-					var i = 0;
+					let i = 0;
+					let async;
```

Comment:
Don't call a variable `async` (future reserved keyword).

Move `async` into the inner scope.